src/apps/util/config.clj

```diff
@@ -102,10 +102,10 @@
   [props config-valid configs]
   "apps.tools.private.time-limit-seconds" (* 24 60 60)) ;; 24 hours
 
-(cc/defprop-optint private-tool-cpu-shares
```

Comment:
I guess the rest aren't either, but I guess this stuff isn't in the `apps.properties.tmpl`? Not sure if we'll need it to be or if we're just worrying about that if/when we do actually need it.